rules.unified

```diff
@@ -55,7 +60,6 @@ KODI_CONFIG_OPTIONS ?= --host=$(DEB_HOST_GNU_TYPE) \
   --enable-nfs \
   --enable-libcec \
   --enable-afpclient \
-  --enable-airtunes \
```

Comment:
this can be reversed, it was me getting around or trying to package last night. I will reverse this and branch it for my own tinkering